google/cloud/completion_queue.h

```diff
@@ -32,7 +32,7 @@ class CompletionQueue;
 
 namespace internal {
 std::shared_ptr<CompletionQueueImpl> GetCompletionQueueImpl(
-    CompletionQueue& cq);
+    const CompletionQueue& cq);
```

Comment:
nit: here and below "east const" s/const CompletionQueue&/CompletionQueue const&/